DCO

```diff
@@ -1,3 +1,6 @@
+SPDX-FileCopyrightText: The RamenDR authors
```

Comment:
This sounds extreme, a copyright on the DCO file? This is the part about SPDX that is confusing me...

Reply:
The DCO already has a copyright in there, we should not be overriding it, correct?

```
Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
660 York Street, Suite 102,
San Francisco, CA 94110 USA
```

Reply:
resue complaing about the DCO files without modification, checking if the issue is 
missing license.